src/components/CreateResponse.js

```diff
@@ -10,7 +10,7 @@ const CreateResponse = ({
   incrementResponses
 }) => {
   return (
-    <div className='ui two cards'>
+    <div className='two column row'>
```

Comment:
We should meet up and figure out some styling guidelines, generally the top level container will have a row, and the children inside it are columns.

https://semantic-ui.com/collections/grid.html